tests/cypress/integration/actions_tasks_objects/case_54_redraw_feature.js

```diff
@@ -0,0 +1,157 @@
+            cy.get('body').trigger('keydown', { keyCode: keyCodeN, shiftKey: true }); // Start redraw the polygon
+            createPolygonShape.pointsMap.forEach((element) => {
+                cy.get('.cvat-canvas-container').click(element.x, element.y - 50);
+            });
+            cy.get('.cvat-canvas-container').trigger('keydown', { keyCode: keyCodeN }).trigger('keyup');
+            cy.get('.cvat_canvas_shape').then(($shape) => {
+                expect($shape.length).to.be.equal(2);
+            });
+            cy.get('.cvat-objects-sidebar-state-item').then(($sidebarItem) => {
+                expect($sidebarItem.length).to.be.equal(2);
+            });
+        });
+
+        it('Draw and redraw a polyline.', () => {
+            cy.createPolyline(createPolylinesShape);
+            cy.get('.cvat-canvas-container').trigger('mousemove', 700, 400);
+            cy.get('#cvat_canvas_shape_3').should('have.class', 'cvat_canvas_shape_activated');
+            cy.get('body').trigger('keydown', { keyCode: keyCodeN, shiftKey: true }); // Start redraw the polyline
+            createPolylinesShape.pointsMap.forEach((element) => {
+                cy.get('.cvat-canvas-container').click(element.x, element.y - 50);
+            });
+            cy.get('.cvat-canvas-container').trigger('keydown', { keyCode: keyCodeN }).trigger('keyup');
+            cy.get('.cvat_canvas_shape').then(($shape) => {
+                expect($shape.length).to.be.equal(3);
+            });
+            cy.get('.cvat-objects-sidebar-state-item').then(($sidebarItem) => {
+                expect($sidebarItem.length).to.be.equal(3);
+            });
+        });
+
+        it('Draw and redraw a point.', () => {
+            cy.createPoint(createPointsShape);
+            cy.get('.cvat-canvas-container').trigger('mousemove', 750, 400);
+            cy.get('body').trigger('keydown', { keyCode: keyCodeN, shiftKey: true }); // Start redraw the point
+            createPointsShape.pointsMap.forEach((element) => {
+                cy.get('.cvat-canvas-container').click(element.x, element.y - 50);
+            });
+            cy.get('.cvat-canvas-container').trigger('keydown', { keyCode: keyCodeN }).trigger('keyup');
+            cy.get('.cvat_canvas_shape').then(($shape) => {
+                expect($shape.length).to.be.equal(4);
+            });
+            cy.get('.cvat-objects-sidebar-state-item').then(($sidebarItem) => {
+                expect($sidebarItem.length).to.be.equal(4);
+            });
+        });
+
+        it('Draw and redraw a tag.', () => {
+            cy.createTag(labelName);
+            cy.get('#cvat-objects-sidebar-state-item-5').trigger('keydown', { keyCode: keyCodeN, shiftKey: true }); // Start redraw the tag
+            cy.get('.cvat-objects-sidebar-state-item').then(($sidebarItem) => {
+                expect($sidebarItem.length).to.be.equal(5);
+            });
```

Comment:
Tag cannot be drawn/redrawn because it does not have a shape. Probably this ``it`` is extra.

Reply:
Removed this ```it```.